packages/client/hmi-client/src/page/project/components/tera-project-overview.vue

```diff
@@ -523,4 +539,27 @@ ul {
 .modal:deep(main) {
 	width: 50rem;
 }
+::v-deep(.asset-button.p-button) {
```

Comment:
I think you can just do `:deep` instead of `::v-deep`. I think `::v-deep` was deprecated

Reply:
Thanks will change